zkevm-circuits/src/evm_circuit/execution/copy_code_to_memory.rs

```diff
@@ -247,7 +244,7 @@ impl<F: FieldExt> ExecutionGadget<F> for CopyCodeToMemoryGadget<F> {
 
         // The number of bytes copied here will be the sum of 1s over the selector
         // vector.
-        let num_bytes_copied = selectors.iter().fold(0, |acc, s| acc + (*s as u64));
+        let num_bytes_copied = selectors.iter().sum::<u8>() as u64;
```

Comment:
For my understanding, is it true that every selector is either 0 or 1 and that num_bytes_copied can't be larger than MAX_COPY_BYTES = 54, so we don't need to worry about overflow here?